src/metabase/api/table.clj

```diff
@@ -133,7 +133,7 @@
   (update-tables! ids body))
 
 
-(def ^:private auto-bin-str (deferred-tru "Auto bin"))
```

Comment:
Hm... I'm not sure about this change.
If you think about it, this string calls to action. Think of it like: 
- What should I do?
- Auto bin (this).

You can see this logic in other options for longitude. It says "Bin every X degrees".

The way I see it and the way it seems to be working in some other examples is:
- While you are choosing the options, it should say "Auto bin" (think of it as "Bin this automatically")
- Later when the option is already selected it should say "Auto binned"

Hope this makes sense?

Reply:
Good point @nemanjaglumac! I like this proposal and seems that I have to change the other message instead (wherever appropriate).